src/tests/bitsets/test_stdlib_bitset_large.f90

```diff
@@ -0,0 +1,1476 @@
+        if ( bits(set0) /= 33 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_0 size properly.'
+        else if ( .not. set0 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                'value properly.'
+        else if ( set0 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                'value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_0 properly into set0'
+        end if
+
+        call set10 % from_string( bitstring_0 // bitstring_0 )
+        if ( bits(set10) /= 66 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_0 // bitstring_0 size properly.'
+        else if ( .not. set10 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                '// bitstring_0 value properly.'
+        else if ( set10 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_0 ' // &
+                '// bitstring_0 value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_0//bitstring_0' // &
+                ' properly into set10'
+        end if
+
+        call set1 % from_string( bitstring_all )
+        if ( bits(set1) /= 33 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_all size properly.'
+        else if ( set1 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else if ( .not. set1 % all() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                'value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_1 properly into set1'
+        end if
+
+        call set11 % from_string( bitstring_all // bitstring_all  )
+        if ( bits(set11) /= 66 ) then
+            error stop procedure // ' from_string failed to interpret ' // &
+                'bitstring_all // bitstring_all size properly.'
+        else if ( set11 % none() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else if ( .not. set11 % any() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else if ( .not. set11 % all() ) then
+            error stop procedure // ' failed to interpret bitstring_all ' // &
+                '// bitstring_all value properly.'
+        else
+            write(*,*) 'from_string transferred bitstring_all // ' // &
```

Comment:
```suggestion
            write(*,*) 'from_string transferred bitstring_all ' // &
```

Reply:
The original code is correct if odd.

Reply:
I understand the code now. Sorry for the misunderstanding.